### PR TITLE
fix: space not found fallback added

### PR DESCRIPTION
**Pull request by Himnashu**
Fixes https://github.com/snapshot-labs/snapshot/issues/2275

Changes proposed in this pull request:
- Show error when space does not exists

### CODE REVIEW

**Review by abhisheknaiidu — APPROVED** [review]
LGTM 🚀

File: src/views/Space.vue [hunk 1]
```diff
@@ -55,7 +55,13 @@ onMounted(() => loadExtentedSpaces([spaceKey.value, sourceSpaceRoute.value]));
         <SpaceSidebarSkeleton />
       </template>
       <template #content-right>
-        <LoadingRow block />
+        <LoadingRow v-if="spaceLoading" block />
+        <template v-else>
+          <BaseBlock :slim="true" class="px-4 py-4 block overflow-hidden">
+            <h2>This account doesn’t exist</h2>
+            <p>Try searching for another.</p>
```
Comment (mktcode) [inline src/views/Space.vue:62]: These need to be translatable strings, using the `$t` function.

File: src/components/SpaceSidebarSkeleton.vue [hunk 2]
```diff
@@ -12,14 +12,25 @@
...
+      />
+      <div
+        class="bg-skin-text h-[32px] lazy-loading mb-1 mt-1 w-[130px] ml-3"
+      />
+      <div
+        class="bg-skin-text h-[32px] lazy-loading mb-1 mt-1 w-[130px] ml-3"
+      />
+      <div
+        class="bg-skin-text h-[20px] lazy-loading mb-1 mt-2 w-[130px] mx-auto"
+      />
```
Comment (mktcode) [inline src/components/SpaceSidebarSkeleton.vue:33]: This doesn't look very nice.

image

I think there should not be any sidebar.

**Review by mktcode — COMMENTED** [review]
It doesn't work for the [settings page](https://bafybeide5us6xsvbaas3wzd3gazsze5o37hhqytu7hqcp6ayho7ldekxnu.on.fleek.co/#/ensasdasdasdad.eth/settings) and [proposals](https://snapshot.org/#/enasdadadds.eth/proposal/0x718c496b04017fb82749b68570d12f32c839f59b9f9433df127f48bf99121eb7).

I think this needs a bit more planning.

**Pull request closed without merging**